package.json

```diff
@@ -33,14 +33,21 @@
     "express": "^4.12.0",
     "express-async-handler": "^1.1.3",
     "express-session": "^1.10.3",
-    "method-override": "^2.3.1",
-    "morgan": "^1.5.1"
+    "method-override": "^3.0.0",
+    "morgan": "^1.9.1"
   },
   "peerDependencies": {
     "bedrock": "^1.5.0",
     "bedrock-server": "^2.0.0"
   },
   "directories": {
     "lib": "./lib"
+  },
+  "devDependencies": {
+    "eslint": "^5.13.0",
```

Comment:
this is technically unncecessary since eslint-config-digitalbazaar provides its own eslint dep.

Possible exception would be if one is going to setup `npm run lint` script as is being done elsewhere.

Reply:
I did setup a lint script, so figured eslint should be a direct dependency.

Reply:
Yeah, I totally get the reasoning behind this, but it's going to be sad when we have to go to eslint.next and change this in all the places.  Seems to me that it would be safe to use the guarantees of eslint being installed provided by eslint-config-digitalbazaar. 

Also, I have no idea what happens if eslint-config-digitalbazaar specifies one eslint version and packages are specifying another.  Seems easy enough to avoid that complication.

Reply:
I think we should be more correct with this in case you are using tools like `pnpm` which defaults to a tree vs flat `node_modules`. In that case you need the dependency else it fails to find the `eslint` binary.

Reply:
@davidlehn in the interest of science, I install pnpm and have found that `pnpm run eslint`on this branch produces the following error.  Are you having different results?
```
bedrock-express@2.1.1-0 lint /home/matt/dev/bedrock-dev/bedrock-express
> eslint 'lib/*.js'

internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module 'prelude-ls'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/home/matt/dev/bedrock-dev/bedrock-express/node_modules/.registry.npmjs.org/eslint/5.13.0/node_modules/optionator/lib/index.js:5:10)
    at Object.<anonymous> (/home/matt/dev/bedrock-dev/bedrock-express/node_modules/.registry.npmjs.org/eslint/5.13.0/node_modules/optionator/lib/index.js:465:4)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
```

Reply:
Worked fine here with:
pnpm 2.25.6
node v10.15.0
npm 6.7.0

Reply:
You already have eslint on your path by chance?

Reply:
Huh, yeah.  If I run the installed eslint direct it fails like you say.  Let's take the conversation off of this issue.